src/dll.py

```diff
@@ -0,0 +1,90 @@
+"""Implementation of Doubly-Linked list with a head and tail."""
+from linked_list import LinkedList
+from linked_list import Node
+
+
+class Dll(object):
+    """Doubly-Linked List class object."""
+
+    def __init__(self):
+        """Doubly-linked list initialization.
+
+        Composed of some attributes from linked-list, and also has a tail.
+        """
+        self._linkedlist = LinkedList()
+        self.head = self._linkedlist.head
+        self._length = self._linkedlist._length
+        self.tail = None
+
+    def push(self, data):
+        """Push node to head of list."""
+        prev_head = self.head
+        new_head = self._linkedlist.push(data)
```

Comment:
`push` shouldn't be returning anything. How are you getting a value back from this operation?